src/screens/notifications/notifications.tsx

```diff
@@ -128,9 +128,11 @@ export class NotificationsComponent extends React.Component<
                 break;
         }
 
-        const { notifications } = this.state;
-        notifications[notificationId].seen = true;
-        this.setState({ notifications });
+        const notifIndex = notifications.findIndex(notif => notif.id === notificationId);
```

Comment:
you don't really need the index, use `find` instead (check this out: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)

Reply:
everything will be more simple :)

Reply:
What do you think about this:

```
this.setState({
            notifications: this.state.notifications.map(notif =>
                notif.id === notificationId
                    ? {
                          ...notif,
                          seen: true
                      }
                    : notif
            )
        });

```

I think this is easier :)